src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -36,8 +44,25 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
         GeFormService.getFormTemplate().then(response => {
             // read the template header info
             if(response !== null && typeof response !== 'undefined') {
-                const { formTemplate } = response;
-                this.initializeForm(formTemplate);
+                let template = response.formTemplate;
+                let permissionErrors = new Array(template.permissionErrors);
```

Comment:
Adding these permissions checks here makes sense for SGE, but this whole block is skipped when loading in Batch context. I think it would be best if we can make this permissions check behavior consistent for both, which probably means moving this to a location where the behavior can be shared.

Reply:
Updated the code to also perform CRUD/FLS checking in the form when loaded in BGE.